src/odemis/gui/comp/overlay/marking_line.py

```diff
@@ -150,8 +150,14 @@ def draw(self, ctx):
 
             # If it's a big number: use at least all the characters needed to show the number (IOW, round to an int).
             # If it's a small number: show 4 significant digits.
-            sig_x = max(4, math.ceil(math.log10(abs(val[0]))))
-            sig_y = max(4, math.ceil(math.log10(abs(val[1]))))
+            def sig_digits(val):
```

Comment:
It does the same thing as 153, 154, right?. Do we make a function to make it readable?

Reply:
No. See commit message. This function now handles correctly `val == 0`, which is a pretty common "corner" case!